version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "1.1.3-SNAPSHOT"
+version in ThisBuild := "1.5.3-SNAPSHOT"
```

Comment:
I don't think this change is meant to be here? IIUC `sbt` will update this file once we `sbt publish`?